taxonomies/Message.ttl

```diff
@@ -436,3 +436,76 @@ ids:LogMessage a owl:Class;
     rdfs:label "Log Message"@en ;
     rdfs:comment "Log Message which can be used to transfer logs e.g. to the clearing house."@en.
 
+
+# App-related Messages
+# ---------------------
+
+ids:AppRegistrationRequestMessage a owl:Class;
+    rdfs:subClassOf ids:RequestMessage ;
+    rdfs:label "App Registration Request Message"@en ;
+    rdfs:comment "Message that asks for registration of a new app to the appStore. Payload contains app data."@en ;
+.
+
+ids:AppRegistrationResponseMessage a owl:Class;
```

Comment:
Since this message is sent for confirmation only, we should rename it to something like `ids:AppRegistrationConfirmedMessage`. The same holds for `ids:AppUpdateResponseMessage` below. This would make it more clear for devs.

Reply:
My intention was to stick to our implicit naming convention, if possible.
____RequestMessage -> ____ResponseMessage. 
This makes it a bit easier to get an understanding about which messages should be used together. 

But you're right. ids:AppRegistrationConfirmedMessage sounds way better :)